src/components/Commands.vue

```diff
@@ -395,33 +401,28 @@ export default class Commands extends Vue {
 
 i.planet {
   &::before {
+    font-size: 25px;
```

Comment:
If you insist on having also a 25px size in the faction button, you must solve the problem of them getting bigger (more height) than regular buttons. I fixed this in player-info by doing `line-height= 1em`, maybe you could add that too in `i.planet` (not in the `::before` part)

Reply:
corrected, leaving the different shadow colors. If you don't like we can revert to your version